packages/core/ui/DrawerWidget.js

```diff
@@ -80,7 +82,11 @@ const DrawerWidget = observer(props => {
             <CircularProgress disableShrink className={classes.drawerLoading} />
           }
         >
-          <LazyReactComponent model={visibleDrawerWidget} session={session} />
+          <LazyWidgetWrapper
```

Comment:
this seems like it likely doesn't have a proper effect of lazily loading the ReactComponent does it?

Reply:
The only component that is lazily loaded is LazyWrapper, but the ReactComponent isn't so I think it is not really doing the lazy loading properly

Reply:
It doesn't code-split the widget component, but lazy loading seems to work, i.e. it will show the fallback until the component is ready.

Reply:
What is it mean that it is waiting for it to be ready? I thought lazy is explicitly for dynamic imports, code splitting and suspense https://reactjs.org/docs/code-splitting.html#reactlazy

Reply:
#1091 is related to this: that's the issue about static files and code splitting in plugin builds